geocoding/example/lib/main.dart

```diff
@@ -49,43 +53,48 @@ class BaseflowPluginExample extends StatelessWidget {
           filled: true,
         ),
       ),
-      home: MyHomePage(title: 'Baseflow ${Globals.pluginName} example app'),
+      home: AppHome(title: 'Baseflow $pluginName example app'),
     );
   }
 
+  /// Method to create MaterialColor based on RGBO Color
   static MaterialColor createMaterialColor(Color color) {
     List strengths = <double>[.05];
     Map swatch = <int, Color>{};
-    final int r = color.red, g = color.green, b = color.blue;
+    final r = color.red, g = color.green, b = color.blue;
 
-    for (int i = 1; i < 10; i++) {
+    for (var i = 1; i < 10; i++) {
       strengths.add(0.1 * i);
     }
-    strengths.forEach((strength) {
-      final double ds = 0.5 - strength;
+    for (var strength in strengths) {
+      final ds = 0.5 - strength;
       swatch[(strength * 1000).round()] = Color.fromRGBO(
         r + ((ds < 0 ? r : (255 - r)) * ds).round(),
         g + ((ds < 0 ? g : (255 - g)) * ds).round(),
         b + ((ds < 0 ? b : (255 - b)) * ds).round(),
         1,
       );
-    });
+    }
     return MaterialColor(color.value, swatch);
   }
 }
 
-class MyHomePage extends StatefulWidget {
-  MyHomePage({Key key, this.title}) : super(key: key);
+/// Application main page
+class AppHome extends StatefulWidget {
+  /// Application main page constructor
+  AppHome({Key key, this.title}) : super(key: key);
 
+  /// Application title
```

Comment:
Adhere to the Effective Dart style guidelines regarding to Doc comments, specifically rule "PREFER starting variable, getter, or setter comments with noun phrases." (https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-variable-getter-or-setter-comments-with-noun-phrases)

```suggestion
  /// The [title] of the application, which is shown in the application's title bar.
```